README.md

```diff
@@ -1,74 +1,177 @@
 # Tellor Data Specifications
 
-This repository contains data specifications for Query IDs. Query IDs are used as unique identifiers for data reported to and retrieved from the Tellor Oracle. In TellorX, `queryID` replaces the legacy `requestID` (tellor3 and earlier). Query ID is the common naming convention across this repository, the [oracle contracts](https://github.com/tellor-io/tellorX/blob/main/contracts/Oracle.sol) themselves, and the [Python telliot client](https://tellor-io.github.io/pytelliot/examples/legacy_query_example/).
```

Comment:
shouldn't we include some info regarding the move from `requestId` to `queryId`?

Reply:
I don't think they have any meaningful relationship.   The move is really from requestID -> Query Descriptor, which I tried to describe, but might not have done well.

Reply:
agreed, it'll be confusing for a day or two, but no need to leave it in